packages/engine/src/scene/systems/ShadowSystem.ts

```diff
@@ -172,7 +172,7 @@ export default async function ShadowSystem(world: World) {
         dropShadowComponent.radius.set(Math.max(sphere.radius * 2, minRadius))
         dropShadowComponent.center.set(groupComponent.value[0].worldToLocal(sphere.center))
       }
-    }, [groupComponent.length, getShadowsEnabled()])
+    }, [groupComponent.length, useShadowsEnabled()])
```

Comment:
let's pull this call out of the dep and have it with the other ones above the effect

Reply:
Ah yes, sorry my mistake. I was rushing so I didn't give enough thought to the last commits.